db/commitlog/commitlog.cc

```diff
@@ -1336,6 +1345,10 @@ void db::commitlog::segment_manager::create_counters(const sstring& metrics_cate
                        sm::description("Holds a size of disk space in bytes used for data so far. "
                                        "A too high value indicates that we have some bottleneck in the writing to sstables path.")),
 
+        sm::make_gauge("disk_slack_bytes", totals.wasted_size_on_disk,
+                       sm::description("Holds a size of disk space in bytes of allocated but unusable data so far. "
+                                       "A too high value indicates that we do not write enough data to each segment.")),
```

Comment:
There's too kinds of slack: padding for block boundary, and slack at the end of the segment. IIUC this is slack at the end. Please clarify it in the description here.

Reply:
Done